content/terms/explanation/filters.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Filters"
+weight: 3
+---
+
+# Filters
+
+Filters solve noise issues in terms versions that cannot be addressed with direct selection or removal of content using CSS selectors or range selectors.
+
+## Why filters are needed
+
+Web pages often contain dynamically generated content or content that cannot be targeted with CSS selectors that creates noise in the archive:
```

Comment:
```suggestion
Web pages often contain dynamically generated content or content that cannot be targeted with CSS selectors that creates noise in the recorded version, for example:
```

Reply:
Or `terms changes history`. It must be consistent with how it is discussed on other pages.